tests/radio_button_tests.py

```diff
@@ -0,0 +1,509 @@
+"""
+Test specific xml tags to ensure they are performing as expected.
+
+This also tests that the validator does in fact load a spec and validate against that spec.
+"""
+
+from lxml import etree
+from unittest import TestCase
+from XmlValidator import ExerciseValidator
+
+
+class RadioButtonValidatorTests(TestCase):
+    """
+    Test that the ExerciseValidator class correctly validates a given XML structure for the radio
+    buttons.
+    """
+
+    def setUp(self):
+        self.exercise_validator = ExerciseValidator()
+
+    def test_validate_with_radio_buttons_without_text(self):
+        """validate: None when radio buttons do not have text, i.e. <button></button>"""
+        good_template_dom = etree.fromstring('''
```

Comment:
The usual practice here is to use bad_template_dom when you are setting up a failing test and good_template_dom when you are setting up a passing test. This clearly indicates that the dom is broken or not.